### DISCUSSION TITLE
ModuleNotFoundError: No module named 'deepdiff' (m1 macOS Big Sur)

### DISCUSSION THREAD

**Discussion by TheMissingPort** [UltimaHoarder/UltimaScraper#899] [General]
Trying to run the code on MacOS

I apologize, new to using GitHub and coding

I have set the default Python to Python 3 

after changing the directory and typing "python3 start_ofd.py", I get the following error:

ModuleNotFoundError: No module named 'deepdiff'

Any tips or suggestions?? Thanks

**Comment by headphonia** [comment]
did you run `pip install -r requirements.txt`?

**Reply by TheMissingPort** [reply]
yes, I had to do pip3 install -r requirements.txt

**Reply by headphonia** [reply]
How about `pip3 install deepdiff`? Also wondering if maybe the deepdiff library isn't available for M1  yet.

**Reply by TheMissingPort** [reply]
That installed, now when I run "python3 start_ofd.py" I get other errors for missing modules. I installed all of them and finally got the .settings file, although only on 5.3

with 6.4.1 it wants me to install dateutil but when I try I get:

ERROR: Could not find a version that satisfies the requirement dateutil (from versions: none)

**Reply by headphonia** [reply]
how about `pip3 install python-dateutil`?

**Reply by TheMissingPort** [reply]
It worked after installing several other modules, it gave me the .settings. 

Thanks!

**Reply by TheMissingPort** [reply]
@headphonia Hey I'm trying to use the latest version of the Onlyfans Scraper, but I'm having trouble with installing "seleniumware"

ERROR: Could not find a version that satisfies the requirement seleniumwire (from versions:none)
Error: No matching distribution found for seleniumwire

Do you have any suggestions? Thank you

**Reply by jaimeggb** [reply]
> It worked after installing several other modules, it gave me the .settings.
> 
> Thanks!

Can you please specify exactly what you did? I am currently in the position you were in back then...
